lib/moon_web/pages/design/popover/content.ex

```diff
@@ -0,0 +1,132 @@
+defmodule MoonWeb.Pages.Design.Popover.Content.Default do
```

Comment:
let's rename it to "MoonWeb.Pages.Design.Popover.Examples.Default"
and please note - u can face some problems with compilation having lot of modules ij a single file. maybe credo, not sure